modules/rhcos-install-iscsi-manual.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-install-iscsi-manual_{context}"]
+= Installing {op-system} manually on an iSCSI boot device
+
```

Comment:
Needs a lead-in sentence. Maybe something like:
You can manually install {op-system} on a device that responds to iSCSI commands, also called an iSCSI target.